src/Common/SipHash.h

```diff
@@ -157,8 +162,8 @@ class SipHash
     void get128(char * out)
     {
         finalize();
-        unalignedStoreLE<UInt64>(out, v0 ^ v1);
```

Comment:
If the old function is not used, delete it.

Reply:
unalignedStoreLE is also used in DoubleDelta,CodecGorilla and compressionCodec, so let's leave it as is.